tests/unit/miniapps/test_sedov.cpp

```diff
@@ -1070,7 +1070,8 @@ class PAMassOperator : public Operator
 
    void Mult(const Vector &x, Vector &y) const
    {
-      ParGridFunction X;
+      // FIXME: why is 'x' being modified here (through 'X')?
+      Vector X;
```

Comment:
Is this has to be done to pass the tests, or is it something that would be good to change independently of this PR?

Reply:
I don't think this affects the `sedov` tests, so it does not need to be addressed right now -- we can just leave the `FIXME` there.

However, modifying the input `const Vector &x` in `Mult` methods, should not be done. This is the first time I see that this is done here, and I want to fix this in the future.